compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/SkiaParagraphIntrinsics.skiko.kt

```diff
@@ -51,7 +51,7 @@ internal class SkiaParagraphIntrinsics(
     private val density: Density,
     private val fontFamilyResolver: FontFamily.Resolver
 ) : ParagraphIntrinsics {
-    val textDirection = resolveTextDirection(style.textDirection)
+    val textDirection = resolveTextDirection(style.textDirection, style.localeList)
```

Comment:
If it's used in a TextField, do you know if it's called for every text change? (I guess the answer is yes)

I'm not sure how valid my concern, it's probably a very rare case:
when the text doesn't contain any strong direction symbols (numbers, signs, punctuation, etc..), then `firstStrongDirectionType ` will have to loop over the entire string.
Would it be valid to limit the number of iterations when we iterate looking for a strong direction symbol?